docs/setup/network-config.md

```diff
@@ -0,0 +1,60 @@
+# Network Recommendations
+This page details the recommendations for network settings to communicate with the Create® 3 via a network. 
+
+## Wi-Fi Networks
+There are several Wi-Fi configurations that must be in place to properly communicate with the Create® 3 via a Wi-Fi network. Below are the currently supported configurations. 
+
+### Supported Configurations
+Authentication: open (no password required) or shared key (password only) 
+
+Frequency: 2.4 GHz
+
+### Troubleshooting
+The ROS 2 node discovery process that is handled by the ROS 2 Middleware (RMW) can be network intensive, but there are several options to mitigate excess traffic and ensure your Create® 3 can operate on your Wi-Fi network. 
+
+#### Disable Multicasting
+Both RMWs (Fast and Cyclone) require multicasting to be enabled on your Wi-Fi network by default. However, multicasting is often disabled on enterprise networks, and the [creators of Fast DDS recommend disabling multicasting](https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/well_known_deployments/well_known_deployments.html) when operating on a Wi-Fi network.   
+
+A complete overview of the multicast disabling process for both RMWs is outlined on the [ROS 2 Middleware Configuration](https://iroboteducation.github.io/create3_docs/setup/xml-config/) page. 
+
+#### Dedicated Network
+In cases where multiple robots will be using a busy network (e.g., university or corporate networks) or the existing network configuration is non-functional, creating a dedicated network for your Create® 3 can mitigate most issues. 
```

Comment:
do we want to mention the possibility of creating an hotspot with a mobile phone?